lib/utils.pm

```diff
@@ -1209,7 +1209,8 @@ sub set_hostname {
             }
 
             for (my $i = 0; $i < 5; $i++) {
-                $state = script_output 'nmcli -w 5 networking connectivity check';
+                # Increase timeout to 60s - poo#169726
+                $state = script_output 'nmcli -w 60 networking connectivity check';
```

Comment:
The repeat loops do not work. If `script_output` fails, it will terminate the test as long as `proceed_on_failure => 1` is not set